dotCMS/src/main/java/com/dotmarketing/cms/urlmap/URLMapAPIImpl.java

```diff
@@ -280,6 +279,7 @@ private String buildContentQuery(
             query.append(this.getHostFilter(context.getHost()));
         }
 
+        query.append(" ");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [StringBuffer (or StringBuilder).append is called consecutively without reusing the target variable.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4748140)